src/components/Map/Map.js

```diff
@@ -64,8 +64,18 @@ const Map = ({ map, permalinkParam }) => {
         center={center}
         zoom={zoom}
         onMapMoved={(evt) => {
-          dispatch(setCenter(evt.map.getView().getCenter()));
-          dispatch(setZoom(evt.map.getView().getZoom()));
+          const newZoom = evt.map.getView().getZoom();
+          const newCenter = evt.map.getView().getCenter();
+          if (
+            (!center && newCenter) ||
+            center[0] !== newCenter[0] ||
+            center[1] !== newCenter[1]
+          ) {
```

Comment:
Why do we need this now and not before ?

Reply:
it is just to avoid dispatching a zoom change if we just pan the map.
But I can let it without condition if you prefer

Reply:
Redux already compare the value and dispatch only when necessary.
maybe it's not the case for the center , in that case do this in the setCenter action function